templates/partials/answer-guidance.html

```diff
@@ -1,13 +1,18 @@
 {% from "components/collapsible/_macro.njk" import onsCollapsible %}
 
 {% call onsCollapsible({
+  "id": "answer-guidance-" ~ answer.id,
   "classes": "u-mt-s",
   "title": _(answer_guidance.schema_item.show_guidance),
   "headingAttributes": {
     "data-ga": "click",
     "data-ga-category": "Answer guidance",
     "data-ga-action": "Open panel",
-    "data-ga-label": _(answer_guidance.schema_item.show_guidance)
+    "data-ga-label": _(answer_guidance.schema_item.show_guidance),
+    "data-qa": "answer-guidance-title"
+  },
+  "contentAttributes": {
+    "data-qa": "answer-guidance-content"
```

Comment:
As there can be more than one answer, this will need to use the answer id as well. 
```suggestion
    "data-qa": "answer-guidance-" ~ answer.id ~ "-content"
```